src/seeder/task-seeder.ts

```diff
@@ -0,0 +1,55 @@
+import { CreateTaskRequest } from './../types/dto/tasks.dto'
+import * as faker from '@faker-js/faker'
+
+import { WorkflowStateResponse } from './../types/dto/workflowStates.dto'
+
+const apiUrl = 'http://localhost:3000'
+
+const token = ''
+
+async function getAllWorkflowStates(): Promise<WorkflowStateResponse[]> {
+  const res = await fetch(`${apiUrl}/api/workflow-states?token=${token}`)
+
+  const data = await res.json()
+
+  return data.workflowStates
+}
+
+async function createTask(payload: CreateTaskRequest) {
+  try {
+    await fetch(`${apiUrl}/api/tasks?token=${token}`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(payload),
+    })
+  } catch (e) {
+    console.log('Error:', e)
+  }
+}
```

Comment:
This is okay, but we might want to insert data into the db directly using prisma instead of hitting the endpoint. This will make it a lot slower.